exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/QueryResultHandler.java

```diff
@@ -351,20 +352,5 @@ public void success(QueryId queryId, ByteBuf buf) {
         }
       }
     }
-
-    @Override
-    public void interrupted(final InterruptedException ex) {
```

Comment:
My bad, I did not consider back pressure. The warning message is misleading. I'll update the warning and user exception message.